src/main/java/com/lootfood/repository/city/CityRepositoryCustomImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.lootfood.repository.city;
+
+import com.lootfood.entity.City;
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.geojson.Point;
+import lombok.RequiredArgsConstructor;
+import org.bson.Document;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+@RequiredArgsConstructor
+public class CityRepositoryCustomImpl implements CityRepositoryCustom {
+    private final MongoTemplate mongoTemplate;
+
+    @Override
+    public City findCityByGeoIntersects(Point point) {
+        FindIterable<Document> result = mongoTemplate.getCollection("city")
```

Comment:
Use a converter to transform a document to City object, as you have done `convertDoc`